### ISSUE TITLE
Import data from DC:Date field

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/Neatline#245]
I also [listed this on the Simile widget issue tracker](https://github.com/scholarslab/nl-widget-Simile/issues/1), but since the fields in the database are added and managed by this plugin, it sees more reasonable to fix this here.

**Comment by TLATER** [comment]
Is there any news on this?

**Comment by jeremyboggs** [comment]
I believe...this is working. @erochest?

**Comment by erochest** [comment]
Nope. I'm pretty sure it's not. Not on my machine, anyway. Definitely will be something we'll be looking at as we dig back into the code, though.
